bindings/python/setup.py

```diff
@@ -93,29 +131,53 @@ def distutils_build():
                             '-DBOOST_ASIO_HAS_STD_CHRONO=1 -DBOOST_EXCEPTION_DISABLE',
                             '-DBOOST_ASIO_ENABLE_CANCELIO', '-DTORRENT_LINKING_SHARED',
                             '-DTORRENT_BUILDING_LIBRARY']
-        )
+    )
     ]
 
     return ext
 
 
-ext = None
+with open('README.rst') as f:
+    readme = f.read()
+
+ext = bjam_build()
+# ext = distutils_build()
 
-if '--help' not in sys.argv \
-        and '--help-commands' not in sys.argv:
-    ext = bjam_build()
-#    ext = distutils_build()
+shutil.copy2('../../COPYING', 'COPYING')
+shutil.copy2('../../LICENSE', 'LICENSE')
 
 setup(
-    name='python-libtorrent',
+    name='libtorrent',
```

Comment:
Well, if packages manually build libtorrent, this probably won't affect them becasue they would need to reconfigure things in any case when they switch to pre-built PyPI version. And it (probably) won't affect distro packages because I think distro package name can be independent from this package, and actual module name is still the same.

Also, someone already uploaded `python-libtorrent` to PyPI some time ago so newer versions can't be published.